aiida/orm/nodes/node.py

```diff
@@ -248,17 +466,6 @@ def __deepcopy__(self, memo):
         """Deep copying a Node is not supported in general, but only for the Data sub class."""
         raise exceptions.InvalidOperation('deep copying a base Node is not supported')
 
-    def initialize(self) -> None:
```

Comment:
I think you'll find removing this method will break lots of things, it shouldn't be done in this PR:

```
aiida/orm/entities.py:
  190          entity = cls.__new__(cls)
  191          entity._backend_entity = backend_entity
  192:         call_with_super_check(entity.initialize)
  193          return entity
  194  

  198          """
  199          self._backend_entity = backend_entity
  200:         call_with_super_check(self.initialize)
  201  
  202      @super_check

aiida/orm/nodes/node.py:
  244          This needs to be called explicitly in each specific subclass implementation of the init.
  245          """
  246:         super().initialize()
  247  
  248          # A cache of incoming links represented as a list of LinkTriples instances

aiida/orm/nodes/data/array/array.py:
  36  
  37      def initialize(self):
  38:         super().initialize()
  39          self._cached_arrays = {}
  40  

aiida/sphinxext/process.py:
  66  
  67      def run(self):
  68:         self.initialize()
  69          return self.build_node_tree()
  70  
```

Reply:
Oh no ignore me, I guess it is specified on the Entity base class

Reply:
I didn't remove it, it is still defined on the `Entity` base class. It just didn't have any specific functionality for the `Node` class anymore.